src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1249,6 +1249,13 @@ public sealed override ImmutableArray<Symbol> GetMembers(string name)
             return ImmutableArray<Symbol>.Empty;
         }
 
+
+        internal override bool HasPossibleWellKnownCloneMethod()
+            // For source symbols, there can only be a valid clone method if this is a record, which is a
+            // simple syntax check. This will need to change when we generalize cloning, but it's a good
+            // heuristic for now.
```

Comment:
My eyes bleed when comments break up code in this way. Possible it's just me. 

Can we put this on top of the method, maybe in a remarks section? 

I'll go back to real feedback now.